src/main/resources/templates/db_scripts/inventory-hierarchy/updateRecordsViewFunctionToSupportAdditionalFields.sql

```diff
@@ -166,6 +166,7 @@ WITH
                                                                  'volume', item.jsonb ->> 'volume',
                                                                  'enumeration', item.jsonb ->> 'enumeration',
                                                                  'chronology', item.jsonb ->>'chronology',
+                                                                 'displaySummary', item.jsonb ->>'displaySummary',
```

Comment:
modified existing view, because schema.json already has declaration to execute this view for Q release:
https://github.com/folio-org/mod-inventory-storage/blob/master/src/main/resources/templates/db_scripts/schema.json#L1154